docs/user-guide/profile.md

```diff
@@ -12,9 +15,44 @@ The profile page allows you to:
  - And change your language settings.
  - This menu also enables you to log out from tmail.
 
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/16c6182e-f434-43e8-ab45-b4c6d757b1ff)
+
 ## Identities
 
-TODO
+Email identities allow you to manage multiple personas from a single email account. This can be useful for personal, professional, or organizational purposes. Each identity can have its own name and signature.
+When you're in Profle page, you can see your identity list
+
+### 1. Create a new identity 
+
+- (1) Click on "Create New Identity" button
+- (2) Enter the name for the identity.
+- (3) Email: Specify the email address associated with this identity. You can select one email from drop-down list. 
+- (4) Reply to:  Specify the email address that will appear on recepient's composer when he reply to your email 
+- (5) BCC: When you compose a new email wiht this identity, the emails in this field will be added to "Bcc" field of composer automatically
+- (6) Customize the signature if needed.
+- (7) Set as default identity: When an identity is set as dedault, its settings such as signature, BCC... will be used when you compose a new email. 
+- (8) Click button Create to save the new identity.
+
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/44f74874-2eff-4aa5-97e5-a267d5a26b71)
+
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/9a02fdae-6aa6-46c2-b599-1bc4d54fb717)
+
```

Comment:
Please add a  note stating that you can include an image in an identity.
Explain that such image is sent on every mail and thus needs to be small to reduce user impact on the planet ;-)